app/src/main/java/dev/arkbuilders/arkmemo/ui/adapters/NotesListAdapter.kt

```diff
@@ -8,62 +8,71 @@ import androidx.fragment.app.FragmentManager
 import androidx.recyclerview.widget.RecyclerView
 import by.kirich1409.viewbindingdelegate.viewBinding
 import dev.arkbuilders.arkmemo.R
-import dev.arkbuilders.arkmemo.databinding.TextNoteBinding
+import dev.arkbuilders.arkmemo.databinding.NoteBinding
+import dev.arkbuilders.arkmemo.models.GraphicNote
+import dev.arkbuilders.arkmemo.models.Note
 import dev.arkbuilders.arkmemo.models.TextNote
 import dev.arkbuilders.arkmemo.ui.activities.MainActivity
-import dev.arkbuilders.arkmemo.ui.activities.replaceFragment
-import dev.arkbuilders.arkmemo.ui.dialogs.NoteDeleteDialogFragment
+import dev.arkbuilders.arkmemo.ui.dialogs.NoteDeleteDialog
+import dev.arkbuilders.arkmemo.ui.fragments.EditGraphicNotesFragment
 import dev.arkbuilders.arkmemo.ui.fragments.EditTextNotesFragment
+import dev.arkbuilders.arkmemo.utils.replaceFragment
 
-class TextNotesListAdapter(private val notes: List<TextNote>): RecyclerView.Adapter<TextNotesListAdapter.NoteViewHolder>() {
+class NotesListAdapter(private val notes: List<Note>):
+    RecyclerView.Adapter<NotesListAdapter.NoteViewHolder>() {
 
     private var activity: MainActivity? = null
     private var fragmentManager: FragmentManager? = null
```

Comment:
We can avoid non-null assertion of `fragmentManager` at line 70 by changing this variable's type to non-null.
The actual `FragmentManager` passed in via `setFragmentManager` is always non-null as per the current usage.